.github/workflows/main.yaml

```diff
@@ -21,16 +41,9 @@ jobs:
     - name: Prepare environment
       run: |
         git config user.name "nuxeo-web-ui-jx-bot" && git config user.email "ui+jx-bot@nuxeo.com"
-        echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
         echo "PACKAGE_VERSION=$(npx -c 'echo "$npm_package_version"')" >> $GITHUB_ENV
 
-    - name: Get pull request version
-      if: github.event_name == 'pull_request'
-      run: |
-        echo "VERSION=$PACKAGE_VERSION-$BRANCH_NAME" >> $GITHUB_ENV
-
     - name: Get prerelease version
-      if: github.event_name == 'push'
```

Comment:
Any particular reason for this change?

Reply:
You mean, to remove the `if` condition? I did it because in the end I removed the `pull_request` event https://github.com/nuxeo/nuxeo-elements/pull/507/files#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3L7

Reply:
We only run the build phase when in push mode